src/agent/rustjail/Cargo.toml

```diff
@@ -11,7 +11,7 @@ serde_derive = "1.0.91"
 oci = { path = "../../libs/oci" }
 protocols = { path ="../../libs/protocols" }
 caps = "0.5.0"
-nix = "0.23.0"
+nix = "0.24.1"
```

Comment:
Why not use a more higher version, `0.24.2` or `0.25.0`?

Reply:
There are quite a few breaking API changes...